fdbmonitor/fdbmonitor.cpp

```diff
@@ -839,7 +843,7 @@ void watch_conf_dir( int kq, int* confd_fd, std::string confdir ) {
 		std::string child = confdir;
 
 		/* Find the nearest existing ancestor */
-		while( (*confd_fd = open( confdir.c_str(), O_EVTONLY )) < 0 && errno == ENOENT ) {
+		while( (*confd_fd = open( confdir.c_str(), O_RDONLY )) < 0 && errno == ENOENT ) {
```

Comment:
I'd prefer to leave this as O_EVTONLY for non-freebsd platforms, as I'd prefer to not have to study the implications of changing it.